packages/core/llm/src/index.ts

```
@@ -0,0 +1,106 @@
+  retryDelay?: number;
+  maxTokens?: number;
+  temperature?: number;
+  metadata?: Record<string, unknown>;
+}
+
+export interface LLMInterface {
+  readonly id: string;
+  readonly name: string;
+
+  getCapabilities(): LLMCapabilities;
+  execute(request: LLMRequest): Promise<LLMResult>;
```

ESLint

[error] 'request' is defined but never used.

(no-unused-vars)